apps/emqx_machine/src/emqx_machine_boot.erl

```diff
@@ -30,12 +30,19 @@
 -export([sorted_reboot_apps/1, reboot_apps/0]).
 -endif.
 
-%% these apps are always (re)started by emqx_machine
+%% These apps are always (re)started by emqx_machine:
 -define(BASIC_REBOOT_APPS, [gproc, esockd, ranch, cowboy, emqx]).
 
-%% If any of these applications crash, the entire EMQX node shuts down
+%% If any of these applications crash, the entire EMQX node shuts down:
 -define(BASIC_PERMANENT_APPS, [mria, ekka, esockd, emqx]).
 
+%% These apps should NOT be (re)started automatically:
+-define(EXCLUDED_APPS, [system_monitor, observer_cli, jq]).
```

Comment:
I believe `jq` is an [optional app](https://github.com/emqx/emqx/blob/c84ee66d801bbe79e9e68e8cad222e778812f84d/mix.exs#L405).

Reply:
@kjellwinblad suggested that we never start JQ supervisor tree because we use NIF interface, rather than port program.